pkg/app/pipedv1/plugin/kubernetes/server.go

```diff
@@ -68,15 +77,24 @@ func (s *server) run(ctx context.Context, input cli.Input) (runErr error) {
 
 	group, ctx := errgroup.WithContext(ctx)
 
-	// TODO: create the piped plugin gRPC client here.
+	if s.pipedPluginServicePort == -1 {
+		input.Logger.Error("piped-plugin-service-port is required")
+		return errors.New("piped-plugin-service-port is required")
+	}
```

Comment:
We can use `MarkFlagRequired` to require the flag 👀 
https://pkg.go.dev/github.com/spf13/cobra#MarkFlagRequired

Reply:
@ffjlabo 
Thank you for notifying me 👍🏻 
Fixed on this commit
https://github.com/pipe-cd/pipecd/pull/5387/commits/b66474b27b40adb0cb0bac05768a76d10c78f836

Reply:
@Warashi 

Then the following line is unnecessary, right?

```go
		pipedPluginServicePort: -1, // default as error value
```

Reply:
oops, that's right. I'll change

Reply:
thanks. fixed on this commit
https://github.com/pipe-cd/pipecd/pull/5387/commits/2d1da06cf08b164298c627d9b58a902adc64ab9b